### PR TITLE
chore: Do not commit generated files

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
This commit removes these generated files from the repository:
- `data/olm-catalog/ssp-operator.clusterserviceversion.yaml`
- `data/crd/ssp.kubevirt.io_ssps.yaml`

Instead of these files, the csv-generator uses the generated `bundle` directory.

This change will be useful in the future when we update `operator-sdk`, because then the generated files will contain a creation timestamp, which would be updated in every new PR.

**Release note**:
```release-note
None
```

### CODE REVIEW

**Review by 0xFelix — COMMENTED** [review]
/approve

File: Dockerfile [hunk 1]
```diff
@@ -52,4 +52,5 @@ USER 1000
 
 # Copy csv generator
 COPY --from=builder /workspace/bin/csv-generator .
+COPY bundle/manifests/ bundle/manifests/
```
[the remark this answers is not in the document]
Comment (akrejcir) [reply Dockerfile:55]: Yes. The issue is that OCP CI builds the image simply using `podman build Dockerfile`, so it does not call `Makefile` to generate missing files.

**Comment by akrejcir** [comment]
Holding until I figure out how to change the build process, so that image can be build in the CI by `podman build Dockerfile`.

/hold

**Pull request closed without merging**